components/loadingBar/theme.css

```diff
@@ -0,0 +1,335 @@
+@import '@teamleader/ui-animations';
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-utilities';
+
+@keyframes slideOut {
+  to {
+    transform: translateX(650%);
+  }
+}
+
+.loading-bar {
+  width: 100%;
+
+  will-change: transform;
+
+  overflow: hidden;
+
+  &-indicator {
+    width: 20%;
+    height: 100%;
+
+    position: relative;
+    left: -25%;
+
+    border-radius: 4px;
+
+    animation: slideOut calc(var(--animation-duration) * 5) ease infinite;
+  }
+}
+
+.is-small {
+  height: var(--spacer-unit);
+}
+
+.is-medium {
+  height: calc(var(--spacer-unit) * 2);
+}
+
+.is-large {
+  height: calc(var(--spacer-unit) * 3);
+}
+
+.is-neutral {
```

Comment:
PP: in the Bullet component we are using PostCSS loops to set the different color - tint variations for the classes, saving a bunch of duplicate code and making it easier to make adjustments.

Reply:
Great idea 👍